gensrc/thrift/InternalService.thrift

```diff
@@ -220,12 +246,10 @@ struct TQueryOptions {
   78: optional i32 spill_encode_level;
   79: optional i64 spill_revocable_max_bytes;
   80: optional bool spill_enable_direct_io;
-  // only used in spill_mode="random"
-  // probability of triggering operator spill
-  // (0.0,1.0)
   81: optional double spill_rand_ratio;
-
   85: optional TSpillMode spill_mode;
```

Comment:
should we move spill_mode into spill_options?

Reply:
yes I have port it to spill_options. 
```
  10: optional double spill_rand_ratio;
  11: optional TSpillMode spill_mode; 
```
we should remove it in 3.4.x